build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 ext {
     wordPressUtilsVersion = 'trunk-68e2995962a5134770e6f9e5b9f258a339e123b2'
     wordPressLoginVersion = '0.15.0'
-    gutenbergMobileVersion = 'v1.80.0'
+    gutenbergMobileVersion = '5040-6094df206c82b139b14634c05ec5b963ddba3f97'
```

Comment:
Friendly reminder: we'll need to update this reference to a newly created alpha tag before merging.